Sources/AppcuesKit/Presentation/Traits/Appcues/TooltipWrapperView.swift

```diff
@@ -11,38 +11,394 @@ import UIKit
+                height: pointerSize.width)
+        case .trailing:
+            // Leading triangle
+            triangleRect = CGRect(
+                x: mainRect.minX - pointerSize.height,
+                y: (mainRect.midY - pointerSize.width / 2 + offsetFromCenter)
+                    .clamped(min: cornerRadius, max: mainRect.maxY - pointerSize.width - cornerRadius),
+                width: pointerSize.height,
+                height: pointerSize.width)
+        }
+
+        let topLeft = CGPoint(x: mainRect.minX + cornerRadius, y: mainRect.minY + cornerRadius)
+        let topRight = CGPoint(x: mainRect.maxX - cornerRadius, y: mainRect.minY + cornerRadius)
+        let bottomLeft = CGPoint(x: mainRect.minX + cornerRadius, y: mainRect.maxY - cornerRadius)
+        let bottomRight = CGPoint(x: mainRect.maxX - cornerRadius, y: mainRect.maxY - cornerRadius)
+
+        // Draw the path clockwise from top left
+        let tooltipPath = UIBezierPath()
+
+        tooltipPath.addArc(withCenter: topLeft, radius: cornerRadius, startAngle: .pi, endAngle: 3 * .pi / 2, clockwise: true)
+
+        // Top triangle
+        if case .bottom = tooltipPosition {
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.minX, y: triangleRect.maxY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.midX, y: triangleRect.minY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.maxX, y: triangleRect.maxY))
+        }
+
+        tooltipPath.addArc(withCenter: topRight, radius: cornerRadius, startAngle: -.pi / 2, endAngle: 0, clockwise: true)
+
+        // Trailing triangle
+        if case .leading = tooltipPosition {
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.minX, y: triangleRect.minY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.maxX, y: triangleRect.midY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.minX, y: triangleRect.maxY))
+        }
+
+        tooltipPath.addArc(withCenter: bottomRight, radius: cornerRadius, startAngle: 0, endAngle: .pi / 2, clockwise: true)
+
+        // Bottom triangle
+        if case .top = tooltipPosition {
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.maxX, y: triangleRect.minY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.midX, y: triangleRect.maxY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.minX, y: triangleRect.minY))
+        }
+
+        tooltipPath.addArc(withCenter: bottomLeft, radius: cornerRadius, startAngle: .pi / 2, endAngle: .pi, clockwise: true)
+
+        // Leading triangle
+        if case .trailing = tooltipPosition {
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.maxX, y: triangleRect.maxY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.minX, y: triangleRect.midY))
+            tooltipPath.addLine(to: CGPoint(x: triangleRect.maxX, y: triangleRect.minY))
+        }
+
+        tooltipPath.close()
+
+        return tooltipPath.cgPath
```

Comment:
holy smokes

Reply:
Just wait for the modifications for pointer edge cases 😅